src/c++/perf_analyzer/report_writer.cc

```diff
@@ -235,6 +246,14 @@ ReportWriter::GenerateReport()
           }
         }
       }
+      if (should_output_llm_metrics_) {
+        if (collector_->IsEmpty()) {
+          throw PerfAnalyzerException(
+              "Attempted to write LLM metrics when profile data is empty.",
+              GENERIC_ERROR);
+        }
+        WriteLlmMetrics(ofs);
```

Comment:
nit: use 'LLM' when you're capitalizing any part of it

Reply:
haha I was actually thinking hard which one to use between the two: `WriteLlmMetrics` vs `WriteLLMMetrics`. But I didn't really like either, and so went with the first one since we already have `WriteGpuMetrics` without caps. Do you think the second one is better?

Reply:
I think seeing a capital L next to a lowercase l is hard to read. The GPU on also should prob change. It's such a small detail. Do whatever you feel like, I'm just writing this here to document my preference.